README.md

```diff
@@ -979,6 +979,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [Krucible](https://usekrucible.com) — Krucible is a platform for creating Kubernetes clusters for testing and development. Free tier accounts come with 25 cluster-hours per month.
   * [Mendix](https://www.mendix.com/) — Rapid Application Development for Enterprises, unlimited number of free sandbox environments supporting unlimited users, 0.5 GB storage and 1 GB RAM per app. Also Studio and Studio Pro IDEs are allowed in free tier.
   * [m3o.com](https://m3o.com) - A cloud platform for API services development. M3O is a fully managed Micro as a Service offering focusing on Go microservices development in the Cloud. Free tier provides enough to run 5 services and collaborate with others.
+  * [Nife.io](https://nife.io) - Developer friendly platform that can build docker containers and deploy serverless applications with CI/CD integartion from Github on any infratsructure, customers on cloud, on prem, private 5G, edge or cloud. Access to 500+ locations, free one app deployment for life. ( Startups and non profits can request for more
```

Comment:
Can you close the parenthesis you opened for Startups line? 
I can't seem to find any technical specifications for the free plan? Can you link to it?